site/homepage/static/css/custom.css

```diff
@@ -116,8 +116,16 @@ pre {
   font-size: 0.9rem;
 }
 
-/*Vertically align adopter logos */
-.customers .owl-wrapper-outer .owl-wrapper {
-  display: flex !important;
-  align-items: center;
+.eclipsefdn-adopters ul {
+  list-style: none;
+}
+
+.eclipsefdn-adopters ul li {
+  display: inline-block;
+  padding: 2%;
+  width: 50%;
+}
+
+.eclipsefdn-adopters ul li a img {
+  width: 100%;
```

Comment:
I have tried this out locally and FMPOV it looks much better at 80%. WDYT?

Reply:
Only reducing it to 80% leads to empty space on the right side. So, I centered the logos additionally in my last commit. WDYT?